lib/api/controllers/funnelController.js

```diff
@@ -216,19 +241,21 @@ function FunnelController (kuzzle) {
           // anonymous user => we throw a 401 (Unauthorised) error
           if (request.context.token.userId === -1) {
             return Promise.reject(new UnauthorizedError(
-              `Unauthorized action [${request.input.resource.index}/${request.input.resource.collection}/${request.input.controller}/${request.input.action}] for anonymous user`)
+              `Unauthorized action [${request.input.index}/${request.input.collection}/${request.input.controller}/${request.input.action}] for anonymous user`)
```

Comment:
I don't remember changing this o_O

According to git blame, this has not changed since 2 months. Maybe a merge problem, I'm fixing that right away.